src/main/java/in/projecteka/gateway/common/cache/RedisCacheAdapter.java

```diff
@@ -63,6 +62,11 @@ public void preDestroy() {
         return retryable(redisCommands.expire(key, 0).then());
     }
 
+    @Override
+    public Mono<String> getIfPresent(String key) {
+        return retryable(get(key));
+    }
+
```

Comment:
why do we need this? what is the difference between get and getIfPresent?

Reply:
We can use get itself. removed getIfPresent.